docs/source/usage_guides/deepspeed_multiple_model.md

```diff
@@ -0,0 +1,215 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contains specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# Using multiple models with DeepSpeed
+
+<Tip warning={true}>
+
+    This guide assumes that you have read and understand the [DeepSpeed usage guide](./deepspeed.md).
+
+</Tip>
+
+## Use cases
+
+There are a few use cases where one may want to run multiple models when using `accelerate` and `DeepSpeed`:
+
+* Knowledge distillation
+* RLHF techniques (see the `trl` repo for [more examples](https://github.com/huggingface/trl))
+* Training multiple models at once.
+
+As it stands, Accelerate has a **very experimental API** to help you accomplish this.
+
+This tutorial will focus on two common use cases:
+
+1. Using `zero2` to train a model, and `zero3` to perform inference for assisting of the training for the `zero2` model (as `zero2` is faster to train on), which is shown in the RLHF example
+2. Training multiple *disjoint* models at once, shown later
+
+## RLHF (multiple models, only one is trained)
```

Comment:
```suggestion
## Multiple models, only one is trained

RLHF is a good example of such use-case
```

The documentation is about many-models-one-trained, not RLHF - so proposing ot re-arrange this